week-1/I-floats/exercise.js

```diff
@@ -1,2 +1,9 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var TotalStudentsAndMentors = numberOfMentors + numberOfStudents;
+var MentorOverTotatal = numberOfMentors / TotalStudentsAndMentors;
+var percentMentors = MentorOverTotatal * 100 + "%";
+var PrecisePecent = Math.round(MentorOverTotatal * 100);
+
+console.log(percentMentors);
+console.log(PrecisePecent);
```

Comment:
All looks good. The only comment I'd make here is that we usually start variables with a lower case letter by convention. So it would be `totalStudentsAndMentors` etc.